docs/design/project-name.md

```diff
@@ -0,0 +1,29 @@
+# Enhancement Title
+
+> [Issue #428](https://github.com/redhat-et/issues/428)
+
+## Summary
+
+Propose a new name for this project: Nexodus.
+
+## Proposal
+
+The current name (Apex) is heavily used, including in a number of places in IT.
+A more unique name for the project will make it more uniquely recognizable, as
+well as more memorable.
+
+* Proposed Project Name: **Nexodus**
+* Proposed GitHub URL: <https://github.com/nexodus-io/nexus>
```

Comment:
@russellb is "nexus" typo in the url or it's correct. If it's correct, than i think we have some popular conflict with it like https://www.sonatype.com/products/nexus-repository

Irrespective of that, thumbs up from me.

Reply:
Typo! Good catch